### PR TITLE
Fix many repeat words

### PR DESCRIPTION
**Pull request by bender2k14**
Here are many fixes for many repeat words.   I only made changes in your posts, your comments, or my comments.  I did not make corrections in the comments of others.

To help in verifying the changes, I put different types of fixes in different commits.

### CODE REVIEW

File: _posts/2016-03-18-functional-architecture-is-ports-and-adapters.html [hunk 1]
```diff
@@ -707,7 +707,7 @@ <h2 id="comments-header">Comments</h2>
...
 			</p>
 			<p>
-				This is one of the many benefits of pure functions: we know, by definition, that they have no side-effects. This property holds not only for the function itself, but for all functions that that pure function calls. This follows from the definition of purity. A pure function can call other pure functions, but it can't call impure functions, because, if it did, it would become impure itself. Haskell enforces this rule, so when you're looking at a pure function, you know that that entire subgraph of your code is pure. Otherwise, it wouldn't compile.
+				This is one of the many benefits of pure functions: we know, by definition, that they have no side-effects. This property holds not only for the function itself, but for all functions that a pure function calls. This follows from the definition of purity. A pure function can call other pure functions, but it can't call impure functions, because, if it did, it would become impure itself. Haskell enforces this rule, so when you're looking at a pure function, you know that that entire subgraph of your code is pure. Otherwise, it wouldn't compile.
```
Comment (ploeh) [inline _posts/2016-03-18-functional-architecture-is-ports-and-adapters.html]: This change look wrong to me. The second _that_ in _that that_ refers back to _the function itself_, whereas the first _that_ is interchangeable with _which_.

File: _posts/2016-03-18-functional-architecture-is-ports-and-adapters.html [hunk 2]
```diff
@@ -707,7 +707,7 @@ <h2 id="comments-header">Comments</h2>
...
 			</p>
 			<p>
-				This is one of the many benefits of pure functions: we know, by definition, that they have no side-effects. This property holds not only for the function itself, but for all functions that a pure function calls. This follows from the definition of purity. A pure function can call other pure functions, but it can't call impure functions, because, if it did, it would become impure itself. Haskell enforces this rule, so when you're looking at a pure function, you know that that entire subgraph of your code is pure. Otherwise, it wouldn't compile.
+				This is one of the many benefits of pure functions: we know, by definition, that they have no side-effects. This property holds not only for the function itself, but for all functions that that a pure function calls. This follows from the definition of purity. A pure function can call other pure functions, but it can't call impure functions, because, if it did, it would become impure itself. Haskell enforces this rule, so when you're looking at a pure function, you know that that entire subgraph of your code is pure. Otherwise, it wouldn't compile.
```
Comment (ploeh) [inline _posts/2016-03-18-functional-architecture-is-ports-and-adapters.html]: Sorry to be sure a nitpicker, but didn't you leave an _a_ that shouldn't be there?

Comment (bender2k14) [reply _posts/2016-03-18-functional-architecture-is-ports-and-adapters.html]: Yea, good catch!  And no sorry needed.  I started us down this nitpicky path ;)

**Comment by ploeh** [comment]
Thank you 👍

**Pull request merged**